### DISCUSSION TITLE
2023/06/my-story-with-OSPP-from-hearing-to-being-selected

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#2] [General]
# 2023/06/my-story-with-OSPP-from-hearing-to-being-selected

6月26日下午3点，OSPP2023 中选结果正式发布，全球共有 1486 人成功申请，最终中选人数为 504 人 鄙人非常荣幸地成为了这 504 个幸运儿的一份子，特撰此文，记录 我与 OSPP 的故事 —— 从听闻到中选 当然，我也希望这篇博客能够吸引更多人参与开源，为开源项目做出自己的贡献  给「杭电助手」打点小广告 本次开源之夏活动，杭州电子科技大学共中选 13 人，其中 8 人来自杭

https://nickxu.me/2023/06/my-story-with-OSPP-from-hearing-to-being-selected.html

**Comment by ywh555hhh** [comment]
谢谢前辈的文章！

**Comment by moment-NEW** [comment]
2026年看到前辈的文章，顿感十分幸运。我是24届的学弟，也是机缘巧合下听说了OSPP这回事。今年的OSPP开始时间十分晚，看着项目列表一直没动静十分不安，好在看到了前辈的文章，顿感之前的迷茫与不安一扫而空。希望今年我也能和前辈一样中选吧！

**Reply by nickxudotme** [reply]
祝你好运😘
